kinder/cmd/kinder/create/cluster/createcluster.go

```diff
@@ -83,14 +79,19 @@ func NewCommand() *cobra.Command {
 	)
 	cmd.Flags().BoolVar(
 		&flags.ExternalEtcd,
-		externalEtcdFlagName, false,
+		"external-etcd", false,
 		"create an external etcd container and setup kubeadm for using it",
 	)
 	cmd.Flags().BoolVar(
 		&flags.ExternalLoadBalancer,
-		externalLoadBalancerFlagName, false,
+		"external-load-balancer", false,
 		"add an external load balancer to the cluster (implicit if number of control-plane nodes>1)",
 	)
+	cmd.Flags().StringSliceVarP(
+		&flags.Volumes,
+		"volume", "v", nil,
```

Comment:
i would keep `volume` only for this instead of `v` too.
kind moved to using `--v` for klog-like verbosity levels.
we might do that in the future too.

Reply:
+1